lib/cursor.js

```diff
@@ -846,21 +857,30 @@ Cursor.prototype.toArray = function(callback) {
  * @param {Cursor~countResultCallback} [callback] The result callback.
  * @return {Promise} returns Promise if no callback passed
  */
+const countSchema = {
+  skip: { type: 'number' },
+  limit: { type: 'number' },
+  maxTimeMS: { type: 'number' },
+  hint: { type: 'string' },
+  readPreference: { type: [ReadPreference, 'object', 'string'] }
```

Comment:
I thought in order to override you had to define the property in the schema? If so, doesn't `session` need to be in here?

Reply:
good catch!